internal/app/machined/internal/phase/install/install.go

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package install
+
+import (
+	"log"
+
+	"github.com/talos-systems/talos/internal/app/machined/internal/mount"
+	"github.com/talos-systems/talos/internal/app/machined/internal/phase"
+	"github.com/talos-systems/talos/internal/app/machined/internal/platform"
+	"github.com/talos-systems/talos/internal/app/machined/internal/runtime"
+	"github.com/talos-systems/talos/pkg/userdata"
+)
+
+// Install represents the Install task.
+type Install struct{}
+
+// NewInstallTask initializes and returns an Install task.
+func NewInstallTask() phase.Task {
+	return &Install{}
+}
+
+// RuntimeFunc returns the runtime function.
+func (task *Install) RuntimeFunc(mode runtime.Mode) phase.RuntimeFunc {
+	switch mode {
+	case runtime.Standard:
+		return task.runtime
+	default:
+		return nil
+	}
+}
+
+func (task *Install) runtime(platform platform.Platform, data *userdata.UserData) (err error) {
```

Comment:
nit, should we change `platform` variable name to something else so it doesn't overlap with package name?

Reply:
Since we don't depend on the platform package any point after, then its fine as is. avoids having a non-descriptive `p` floating around. But happy to change if you feel strongly about it.